releasenotes/notes/add-2q-fractional-gates-to-unitarysynthesis-pass-f66eee29903f5639.yaml

```
@@ -0,0 +1,10 @@
+---
+features_synthesis:
+  - |
+    Add a :class:`.TwoQubitControlledUDecomposer` that decomposes any two-qubit unitary
+    in terms of basis two-qubit fractional gates, such as :class:`.RZZGate`
+    (or two-gates gates which are locally equivalent to :class:`.RZZGate` up to single qubit gates).
+features_transpiler:
+  - |
+    Add two-qubit fractional basis gates, such as :class:`.RZZGate`, to the
+    :class:`.UnitarySynthesis` transpiler pass.
```

Same comment as above, we should add an example usage.

```suggestion
    Added support for two-qubit fractional basis gates, such as :class:`.RZZGate`, to the
    :class:`.UnitarySynthesis` transpiler pass. The decomposition is done using the :class:`.TwoQubitControlledUDecomposer`, and supports both standard and custom basis gates.
```

---

done in [55556f8](https://github.com/Qiskit/qiskit/pull/13568/commits/55556f8f4762d960391311a76d2ecc4ee2d3222c)